src/main/scala/scalafix/sbt/ScalafixPlugin.scala

```diff
@@ -89,7 +89,13 @@ object ScalafixPlugin extends AutoPlugin {
           "Defaults to a wrapper around `sbt.Logger`."
       )
 
-    val scalafixSemanticdb: ModuleID =
+    val scalafixAllowDynamicFallback: SettingKey[Boolean] =
+      settingKey[Boolean](
+        "Control whether the latest version of scalafix may be downloaded instead of failing when " +
+          "scalafix-interfaces is not explicitly provided. Off by default because non-deterministic."
+      )
```

Comment:
this was first a private key used by scalafixEnable, but I thought we could make it public for power users, without necessarily documenting it